frontends/web/.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": ["react-app"],
+  "rules": {
+    "jsx-a11y/anchor-is-valid": 0,
```

Comment:
There are some linting problems, but they are non-breaking changes. I disabled that rule just for making a more simple PR.